NuGet.Config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetrelease/api/v2" />
+    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetvnext/api/v2" />
```

Comment:
This is not coming from me but from the current dev branch I was based on and I'm struggling getting rid of it. 
Anyways, I'm creating a new branch from release to create a clean patch. So I'm closing this pull request, and I'll submit a cleaner one.